### PR TITLE
Add more tests to types/vote.go

### PR DESCRIPTION
**Pull request by ricardohsd**
Improve test coverage of `types/vote.go`

### CODE REVIEW

File: types/vote_test.go [hunk 1]
```diff
@@ -45,18 +53,30 @@ func TestVoteSignable(t *testing.T) {
...
-	str := exampleVote().String()
-	expected := `Vote{56789:616464720000 12345/02/2(Precommit) 686173680000 {<nil>} @ 2017-12-25T03:00:01.234Z}`
-	if str != expected {
-		t.Errorf("Got unexpected string for Proposal. Expected:\n%v\nGot:\n%v", expected, str)
+	tc := []struct {
+		name string
+		in   string
+		out  string
+	}{
+		{"Precommit", examplePrecommit().String(), `Vote{56789:616464720000 12345/02/2(Precommit) 686173680000 {<nil>} @ 2017-12-25T03:00:01.234Z}`},
```
Comment (odeke-em) [inline types/vote_test.go]: Nice!

**Comment by melekes** [comment]
@ricardohsd could you please rebase against current develop and re-push?

**Comment by ebuchman** [comment]
No worries, merged

**Comment by ricardohsd** [comment]
@melekes Sorry. Just saw your comment. I was off due to holidays.

**Pull request merged**